CONTRIBUTING.md

```diff
@@ -57,3 +58,15 @@ This happens when godotenv was not (or incorrectly) installed. Install it by run
 #### 3. mage watch throws 'too many open files' error
 
 macOS has a small limit on how many files can be open at the same time. This limit is usually OK for most users, but developers tools usuage require a larger limit. [Learn how to resolve this](https://www.macobserver.com/tips/deep-dive/evade-macos-many-open-files-error-pushing-limits/).
+
+#### 4. mage isn't found even after installing (Windows)
```

Comment:
This is a good one! But it's not specific to Windows. If we make the text OS-agnostic, we can add this.